pkg/dashboard/apiserver/event/event.go

```diff
@@ -97,6 +106,116 @@ func (s *Service) list(c *gin.Context) {
 	c.JSON(http.StatusOK, events)
 }
 
+// @Summary cascadeFetchEventsForWorkflow list all events for Workflow and related WorkflowNode.
+// @Description list all events for Workflow and related WorkflowNode.
+// @Tags events
+// @Produce json
+// @Param namespace query string false "The namespace of the object"
+// @Param object_id query string false "The UID of the Workflow"
```

Comment:
Looks like "object_id" is useless?

Reply:
🙈 The comments for swagger need updates.  I am going to fix that.